package.json

```diff
@@ -53,7 +55,7 @@
     "defi-legos"
   ],
   "dependencies": {
-    "@openzeppelin/contracts": "4.4.0"
+    "@openzeppelin/contracts": "v3.4.0"
```

Comment:
Why was this downgraded?

Reply:
SafeMath version used in FlashLoanBase is different in 4.4.0. The one in v3.4.0 is required.